config/bench/config_files/spark-1.6.1_conf_template/spark-defaults.conf

```diff
@@ -25,3 +25,5 @@
 # spark.serializer                 org.apache.spark.serializer.KryoSerializer
 # spark.driver.memory              5g
 # spark.executor.extraJavaOptions  -XX:+PrintGCDetails -Dkey=value -Dnumbers="one two three"
+
```

Comment:
We are not using this configuration folders as they where the ones we used when testing spark. Only folders spark-{1,2}.x_conf_template are being used. You can savely delete these folders.

Reply:
Nice to know, I was wondering if I actually needed this. I change this and add a commit to remove this directory.